### PR TITLE
Add a `--skip-yarn` option on the `ReactSetupGenerator`:

### PR DESCRIPTION
**Pull request by Edouard-chin**
- ### Context

  For the Gestalt project, each Rails application will be boostraped
  using a tool to generate Shopify's flavoured app. The application
  will be provisioned in the cloud, in this context installing JS
  dependencies of the app is useless. What we really need is to add
  the JS dependencies as part of the yarn package.json and lock
  the version in the yarn lockfile.

  ### Problem

  Yarn doesn't have a way to add new dependencies to a project
  without downloading them (i.e modify only the package.json/yarn.lock)
  files. Downloading those dependencies takes a realllly long time,
  basically 50% of the time to boostrap an app is dedicated to
  downloading JS dependencies.

  ### Solution

  Introduce a flag on this generator to not add/install JS
  dependencies.
  Having that flag will allow us to control programatically
  when JS dependencies get installed.

### CODE REVIEW

**Comment by iaserrat** [comment]
Not sure I'm the right reviewer for this. What is this?

**Comment by Edouard-chin** [comment]
My bad @iaserrat wrong ping

**Comment by Edouard-chin** [comment]
Oops sorry I didn't see there was a CHANGELOG entry in this repo
